packages/suite/src/views/wallet/coinmarket/common/CoinmarketLayoutNew/CoinmarketLayoutNavigation/CoinmarketLayoutNavigation.tsx

```diff
@@ -1,18 +1,39 @@
 import styled from 'styled-components';
-import { useDevice } from 'src/hooks/suite';
+import { useDevice, useSelector } from 'src/hooks/suite';
 import { FirmwareType } from '@trezor/connect';
 import CoinmarketLayoutNavigationItem from './CoinmarketLayoutNavigationItem';
+import { Divider } from '@trezor/components';
+import { spacings } from '@trezor/theme';
+import { useMemo } from 'react';
+import regional from 'src/constants/wallet/coinmarket/regional';
+import { getIsTorEnabled } from 'src/utils/suite/tor';
 
 const List = styled.div`
     display: flex;
     flex-wrap: wrap;
     align-items: center;
 `;
 
+const SeparatorWrapper = styled.div`
+    height: 42px;
+`;
+
 const CoinmarketLayoutNavigation = () => {
     const { device } = useDevice();
     const isBitcoinOnly = device?.firmwareType === FirmwareType.BitcoinOnly;
 
+    const torStatus = useSelector(state => state.suite.torStatus);
+    const isTorEnabled = getIsTorEnabled(torStatus);
+    const country = useSelector(
+        state =>
+            state.wallet.coinmarket.buy.buyInfo?.buyInfo?.country ??
```

Comment:
this is interesting 🤔

`buyInfo?.buyInfo?`

Reply:
yeah, it looks weird, but it's correct (I'm using already defined state)